models/source/im-bm-ci/im_msrtd_format.sql

```diff
@@ -0,0 +1,20 @@
+{{
+  config({
+    "materialized" : "table",
+    "post-hook" : [
+      "grant select on table {{this}} to group non_gl_read_only"
+      ]
+    })
+}}
+
+select
+    udf_msrtd_format_id::int msrtd_format_id,
+    udf_format_desc as format_desc,
+    nullif(udf_sub_unit_format, '')::int as sub_unit_format,
+    nullif(udf_case_eq, '')::float as case_eq,
+    nullif(udf_hi, '')::int as hi,
+    nullif(udf_ti, '')::int as ti,
+    nullif(udf_upp, '')::int as units_per_pack,
+    udf_stack_pallets as stackable,
```

Comment:
Can we convert this to a boolean field, rather htan a Y/N varchar? You can do this with something like this:

udf_stack_pallets = 'Y' as stackable